Getting this Mono error. Doesn't seem to be wrapping. (I don't have `mono` installed system-wide)

![image](https://github.com/user-attachments/assets/0c55dfa1-26d0-45dd-9dc0-9ed002fd919c)

![Screenshot_20241230_074823](https://github.com/user-attachments/assets/6c75b8c9-37a4-44a4-ab9d-2d4deabe6a06)
[Compared to another 'wrapped' package.](https://github.com/user-attachments/assets/00ba9322-dfe2-4a64-b6d8-3dd9434217a7) Maybe it needs `makeWrapper`? 

Also seeing an error with Mesa in the logs, could be harmless.

```
[07:39:37][INFO] Start of Outer Wilds Mod Manager v0.15.0
[07:39:37][DEBUG] Setup window state
[07:39:37][INFO] File Watcher Setup On Standby
MESA-LOADER: failed to open dri: /run/opengl-driver/lib/gbm/dri_gbm.so: cannot open shared object file: No such file or directory (search paths /run/opengl-driver/lib/gbm, suffix _gbm)
Failed to create GBM device for DRM node: /dev/dri/renderD128: No such file or directory
[07:39:38][DEBUG] Reading Config From /home/keenan/.local/share/ow-mod-man/settings.json
[07:39:38][INFO] File Watcher: Watching Filesystem
[07:39:38][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
[07:39:38][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:38][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:38][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:38][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:38][DEBUG] Fetching Remote DB At https://ow-mods.github.io/ow-mod-db/database.json
[07:39:38][DEBUG] Success, Constructing Remote Mod Map
[07:39:43][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:43][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:43][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:44][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:44][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:44][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:51][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
[07:39:51][DEBUG] Begin construction of local db at /home/keenan/.local/share/OuterWildsModManager/OWML
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/JohnCorby.VanillaFix/manifest.json
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/Raicuparta.QuantumSpaceBuddies/manifest.json
[07:39:51][DEBUG] Loading Mod With Manifest: /home/keenan/.local/share/OuterWildsModManager/OWML/Mods/_nebula.MenuFramework/manifest.json
MESA-LOADER: failed to open dri: /run/opengl-driver/lib/gbm/dri_gbm.so: cannot open shared object file: No such file or directory (search paths /run/opengl-driver/lib/gbm, suffix _gbm)
Failed to create GBM device for DRM node: /dev/dri/renderD128: No such file or directory
[07:39:52][DEBUG] Writing Config To /home/keenan/.local/share/ow-mod-man/settings.json
[07:39:52][ERROR] Error Received From Frontend: Can't Start Game: Failed to launch game: Os { code: 2, kind: NotFound, message: "No such file or directory" }. Is Mono Installed?
[07:39:52][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
[07:39:53][DEBUG] Reading Config From /home/keenan/.local/share/ow-mod-man/settings.json
[07:39:53][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
[07:39:53][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
MESA-LOADER: failed to open dri: /run/opengl-driver/lib/gbm/dri_gbm.so: cannot open shared object file: No such file or directory (search paths /run/opengl-driver/lib/gbm, suffix _gbm)
Failed to create GBM device for DRM node: /dev/dri/renderD128: No such file or directory
[07:39:58][DEBUG] Writing Config To /home/keenan/.local/share/ow-mod-man/settings.json
[07:39:58][ERROR] Error Received From Frontend: Can't Start Game: Failed to launch game: Os { code: 2, kind: NotFound, message: "No such file or directory" }. Is Mono Installed?
[07:39:58][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
[07:39:58][DEBUG] Reading Config From /home/keenan/.local/share/ow-mod-man/settings.json
[07:39:58][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
[07:39:58][DEBUG] Fetching Alert At: https://raw.githubusercontent.com/ow-mods/ow-mod-db/source/alert-v2.json
thread '<unnamed>' panicked at /build/owmods-gui-0.15.0-vendor.tar.gz/glib/src/thread_guard.rs:88:9:
Value accessed from different thread than where it was created
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread '<unnamed>' panicked at /build/owmods-gui-0.15.0-vendor.tar.gz/glib/src/thread_guard.rs:107:9:
Value dropped on a different thread than where it was created
stack backtrace:
```

--- a/pkgs/by-name/ow/owmods-gui/package.nix
+++ b/pkgs/by-name/ow/owmods-gui/package.nix
@@ -1,0 +1,122 @@
+{
+  lib,
+  libsoup_3,
+  dbus,
+  glib,
+  glib-networking,
+  librsvg,
+  webkitgtk_4_1,
+  pkg-config,
+  wrapGAppsHook3,
+  makeDesktopItem,
+  copyDesktopItems,
+  rustPlatform,
+  buildNpmPackage,
+  fetchFromGitHub,
+  mono,
+  wrapWithMono ? true,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "owmods-gui";
+  version = "0.15.0";
+
+  src = fetchFromGitHub {
+    owner = "ow-mods";
+    repo = "ow-mod-man";
+    tag = "gui_v${version}";
+    hash = "sha256-rTANG+yHE8YfWYUyELoKgj4El+1ZW6vI9NkgADD40pw=";
+  };
+
+  cargoHash = "sha256-rFkh2G7kFuQI7nlZIwaqvt7x9bKLqmWU21YwZu2+wUA=";
+
+  buildFeatures = [
+    "tauri/custom-protocol"
+  ];
+
+  nativeBuildInputs = [
+    pkg-config
+    copyDesktopItems
+    wrapGAppsHook3
+  ];
+
+  buildInputs = [
+    dbus
+    libsoup_3
+    glib
+    librsvg
+    glib-networking
+    webkitgtk_4_1
+  ];
+
+  buildAndTestSubdir = "owmods_gui/backend";
+
+  preFixup = lib.optionalString wrapWithMono "gappsWrapperArgs+=(--prefix PATH : '${mono}/bin')";
+
+  postPatch =
+    let
+      frontend = buildNpmPackage {
+        inherit version;
+
+        env.VITE_VERSION_SUFFIX = "-nix";
+
+        pname = "owmods-gui-ui";
+        src = "${src}/owmods_gui/frontend";
+
+        packageJSON = "${src}/owmods_gui/frontend/package.json";
+        npmDepsHash = "sha256-h6e+hQzd52G3XtufioEYlBuXNu6I+ZTQcNgJaQdaAck=";
+
+        postBuild = ''
+          cp -r ../dist/ $out
+        '';
+        distPhase = "true";
+        dontInstall = true;
+        installInPlace = true;
+        distDir = "../dist";
+
+        meta = {
+          description = "Web frontend for the Outer Wilds Mod Manager";
+          homepage = "https://github.com/ow-mods/ow-mod-man/tree/main/owmods_gui/frontend";
+          license = lib.licenses.gpl3Plus;
+        };
+      };
+    in
+    ''
+      substituteInPlace owmods_gui/backend/tauri.conf.json \
+        --replace-fail '"frontendDist": "../dist"' '"frontendDist": "${frontend}"'
+    '';
+
+  postInstall = ''
+    install -DT owmods_gui/backend/icons/128x128@2x.png $out/share/icons/hicolor/256x256@2/apps/outer-wilds-mod-manager.png
+    install -DT owmods_gui/backend/icons/128x128.png $out/share/icons/hicolor/128x128/apps/outer-wilds-mod-manager.png
+    install -DT owmods_gui/backend/icons/32x32.png $out/share/icons/hicolor/32x32/apps/outer-wilds-mod-manager.png
+
+    mv $out/bin/owmods_gui $out/bin/outer-wilds-mod-manager
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "outer-wilds-mod-manager";
+      exec = "outer-wilds-mod-manager %u";
+      icon = "outer-wilds-mod-manager";
+      desktopName = "Outer Wilds Mod Manager";
+      categories = [ "Game" ];
+      comment = "Manage Outer Wilds Mods";
+      mimeTypes = [ "x-scheme-handler/owmods" ];
+    })
+  ];
+
+  meta = {
+    description = "GUI version of the mod manager for Outer Wilds Mod Loader";
+    homepage = "https://github.com/ow-mods/ow-mod-man/tree/main/owmods_gui";
+    downloadPage = "https://github.com/ow-mods/ow-mod-man/releases/tag/gui_v${version}";
+    changelog = "https://github.com/ow-mods/ow-mod-man/releases/tag/gui_v${version}";
+    mainProgram = "outer-wilds-mod-manager";
+    platforms = lib.platforms.linux;
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [
+      bwc9876
+      locochoco
+      spoonbaker
+    ];
+  };
+}

